### ISSUE TITLE
Files Reload 

### ISSUE THREAD

**Issue by Sulemanali511** [humy2833/FTP-Simple#234]
whenever i create a file on server manually without using this extension, there is no command for reloading the whole directory which helps the coder for viewing which file is new in directory. for example on same project more than one developer working together and one of them created a file on server other will unknown from it until the person who created tell the other... kindly update this extension with this feature like directory sync.

**Comment by LEXWORTHY** [comment]
I have this issue as well.

**Comment by bm322741** [comment]
I would like the feature to just reload an open file. I have log files which I'm constantly closing & reopening. A reload would save a lot of time.

**Comment by lefuturiste** [comment]
Yes, same issue when you delete a file on a server via ssh for example, the "refresh" button in the file explorer side bar doesn't do anything

**Comment by Sulemanali511** [comment]
sorry

السلام عليكم و رحمة الله و بركاته












Best Regards
*Suleman Ali*
*Department Representative of Computer Science*
*CR of BSCS 8th semester*
*+92302-7655876*
